open-build-service/create_archive

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+version=$(cat conf-version)
+if [ ! -f  create_archive ] ; then
+	echo "create_archive not found in current directory" 1>&2
+	exit 1
+fi
+cd ../..
+echo "Preparing notqmail-"$version""
+mkdir -p $HOME/stage
+/bin/cp -rp notqmail $HOME/stage/notqmail-"$version"
+cd $HOME/stage/notqmail-$version
+echo "Cleaning notqmail-"$version""
+make -s clean >/dev/null 2>&1
+# $HOME/stage/
+cd ..
+echo "Archiving notqmail-"$version".tar.xz in `pwd`"
+tar \
+	--exclude="notqmail-$version/.git" \
+	--exclude="notqmail-$version/open-build-service"  \
+	--exclude="notqmail-$version/autobuilds"  \
```

Comment:
Please add a comment pointing to other places that we need to exclude these files (and comments in those places, too). Or, if we can just do it the same way as [our release process](https://notqmail.org/doing-releases/) states, which is currently believed to be the below, that's good enough:

```sh
git archive --prefix=notqmail-${RELEASE}/ -o notqmail-${RELEASE}.tar notqmail-${RELEASE}
```

Reply:
This script is not needed. Tried git archive and it does the same thing except for inclusion of open-build-service directory (which is not needed for the binary buld)

Reply:
Will remove this script in the next commit

Reply:
Instead of removing the script, I have updated it with the git archive command in commit #[3eaf213](https://github.com/notqmail/notqmail/pull/76/commits/3eaf2138ce40da0831bfc90852faa8b7c2a2c4ff)